### PR TITLE
LPS-165583 Add Translation and fix date format

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/i18n/en_US.ts [hunk 1]
```diff
@@ -122,6 +123,7 @@ export default {
 	'current-requirements': 'Current Requirements',
 	'current-subcomponents': 'Current Sub Components',
 	'current-teams': 'Current Teams',
+	'data-last-modified': 'Data last modified',
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/i18n/en_US.ts:126]:
```suggestion
	'date-last-modified': 'Date last modified',
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/i18n/en_US.ts [hunk 2]
```diff
@@ -30,6 +30,7 @@ export default {
 	'add-suites': 'Add Suites',
 	'add-user-account': 'Add User Account',
 	'add-x': 'Add {0}',
+	'all-issue-found': 'All issue found',
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/i18n/en_US.ts:33]: This translation looks weird, is that right?

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/Export.tsx [hunk 3]
```diff
@@ -13,6 +13,7 @@
  */
 
 import ClayTable from '@clayui/table';
+import dayjs from 'dayjs';
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/Export.tsx:16]: Use this import instead, the ../../ may be different

We have **dayjs** configured there

```suggestion
import dayjs from '../../util/date';
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
@welitonpe see

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**